Userland/Libraries/LibWeb/Loader/ResourceLoader.cpp

```diff
@@ -97,11 +100,13 @@ static String sanitized_url_for_logging(AK::URL const& url)
     return url.to_string();
 }
 
+#ifdef __serenity__
 static void emit_signpost(String const& message, int id)
 {
     auto string_id = perf_register_string(message.characters(), message.length());
     perf_event(PERF_EVENT_SIGNPOST, string_id, id);
 }
+#endif
```

Comment:
Could this ifdef be the inside the function? This way we wouldn’t need to wrap calls to this function with ifdefs